src/modules/rabbitmq/elastic-updates/elastic-updates-events.service.ts

```diff
@@ -116,12 +116,12 @@ export class ElasticUpdatesEventsService {
       }
 
       nft = await this.assetByIdentifierService.getAsset(identifier);
-      collectionTypes.collection = nft.type;
 
       if (!nft || Object.keys(nft).length === 0) {
         continue;
       }
 
```

Comment:
collection = type? can you rename maybe?

Reply:
It seems it was a logic error :man_facepalming: ; fixed with `collectionTypes[collection]`